README.md

```diff
@@ -12,22 +12,34 @@ for help getting started.
 
 Visit the [Servo Project page](https://servo.org/) for news and guides.
 
-## Prerequisites
-
-On OS X (homebrew):
+## Setting up your environment
+
+Please select your operating system:
+* [OSX (homebrew)](#osxHome)
+* [OSX (MacPorts)](#osxPorts)
+* [OSX 10.11 (El Capitan) with openssl](#osxCapitan)
+* [Debian-based Linuxes](#debian)
+* [Fedora](#fedora)
+* [Arch Linux](#archLinux)
+* [Gentoo Linux](#gentooLinux)
+* [Microsoft Windows](#windows)
+* [Android](#android)
+
+<a name="osxHome"></a>
```

Comment:
Rather than using `a` elements, I'd use headers; github will generate ids automatically.

Also, I'd collapse the three OSX sections into one.